Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -250,6 +250,25 @@ def fetch_incidents(client, last_run, first_fetch_time, event_type_filter, threa
 
     return next_run, incidents
 
+def get_forensics(client, threat_id,server_url,api_version,verify_certificate,service_principal,secret):
+    forensic_url = "{}/{}/forensics?threatId={}".format(server_url, api_version,threat_id)
+    method = "GET"
+
+    res = requests.request(
```

Comment:
instead of using the requests in this function, add `get_forensics` function to the client class. In that class use `self.http_request` like other functions do. http_request will take care of status_code and returning json